firefox_ui_tests/functional/private_browsing/test_about_private_browsing.py

```diff
@@ -34,17 +34,18 @@ def testCheckAboutPrivateBrowsing(self):
                                                        'p.showNormal')
             self.assertEqual(status_node.text, description)
 
-            access_key = self.browser.get_localized_entity(
-                'privatebrowsingpage.openPrivateWindow.accesskey')
+            # TODO: Augment send_shortcut() to DOMElement and let Marionette return
+            # such an object instead of HTMLElement, so it can be send to the element
 
             # Send keys to the top html element.
-            top_html = self.marionette.find_element(By.TAG_NAME, 'html')
-            top_html.send_keys(self.keys.SHIFT, self.keys.ACCEL, access_key)
+            # top_html = self.marionette.find_element(By.TAG_NAME, 'html')
+            # top_html.send_keys(self.keys.SHIFT, self.keys.ACCEL, access_key)
```

Comment:
I wanted to leave it in here so I know what to revert later. I'm not sure right now if sending the access key to the window is the final right approach. Given that I brought back the keys module, I can revert that one for now.